src/library_dylink.js

```diff
@@ -743,11 +745,11 @@ var LibraryDylink = {
     }
 
     // We don't care about RTLD_NOW and RTLD_LAZY.
-    var jsflags = {
-      global:   Boolean(flags & {{{ cDefine('RTLD_GLOBAL') }}}),
-      nodelete: Boolean(flags & {{{ cDefine('RTLD_NODELETE') }}}),
+    jsflags.global = Boolean(flags & {{{ cDefine('RTLD_GLOBAL') }}});
+    jsflags.nodelete = Boolean(flags & {{{ cDefine('RTLD_NODELETE') }}})
```

Comment:
it's a little weird to modify the jsflags input object - the caller may notice that. how about using locals for this?

Reply:
The goal here is to combine the incoming JS flags with the incoming non-JS flags.. into a new jsflags object that can be passed down the the underlying function.   I could create an entirely new JS object based on both but that seems like more code (i'm not sure sure how to best to that).

Reply:
I guess since we are the only caller, maybe it's ok either way, but I'd prefer to create a new object here, and copy things from the passed-in object onto it. That is safer going forward.

Reply:
(or, the object fields could each be locals perhaps?)

Reply:
The problem is that then we need to remember to update this code every time we change the number of fields.. either than or use Object.assign which I don't think we are able to use yet?

Reply:
Done!